js/main.js

```diff
@@ -0,0 +1,68 @@
+const PHOTO_COUNT = 25;
+const MIN_LIKES = 15;
+const MAX_LIKES = 200;
+const MAX_COMMENTS = 30;
+const MAX_COMMENT_ID = 100;
```

Comment:
Попробуй догадаться, что убивало страницу
```suggestion
const MAX_COMMENT_ID = 1_000_000;
```